CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
+
+## Pre commits
+
+We use precommit hooks to ensure code style and format.
+
+* Install precommit from pip
+
+  ```
+  pip install pre-commit
+  ```
+
+* Now after each commit, the style hooks will run and auto format the code.
```

Comment:
We also need to have the hooks installed (and missing a dash). We could also mention how to run the hooks without an actual commit:

----------

## Pre-commit

Install pre-commit from pip:

  ```bash
  pip install pre-commit
  pre-commit install
  ```

 Now after each commit, the style hooks will run and auto format the code.

  You can also manually run the pre-commit hooks without a commit with `pre-commit run -a`